src/cdk/table/sticky-styler.ts

```diff
@@ -125,6 +132,20 @@ export class StickyStyler {
           }
         }
       }
+
```

Comment:
It would be good to add a brief comment here that touches on what the listener is for / why it's notified.